ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -261,6 +261,9 @@ static ggml_cuda_device_info ggml_cuda_init() {
         GGML_LOG_INFO("  Device %d: %s, %s (0x%x), VMM: %s, Wave Size: %d\n",
                       id, prop.name, prop.gcnArchName, info.devices[id].cc & 0xffff,
                       device_vmm ? "yes" : "no", prop.warpSize);
+#elif defined(GGML_USE_MUSA)
+        info.devices[id].smpbo = prop.sharedMemPerBlockOptin;
+        info.devices[id].cc = 100*prop.major + 10*prop.minor;
```

Comment:
You can revert this change since it's identical to lines 268-269.

Reply:
Hi @yeahdongcn , as we aligned before, we need to keep this for future refining of the MUSA CC, similar to how `HIP` handles this.

Also added back the `cc` info log.

Reply:
Just add a comment explaining why this is duplicated to avoid confusion in the future.

Reply:
> Just add a comment in the code explaining why this is duplicated to avoid confusion in the future.

Hi @ggerganov , I have added comment for the MUSA .cc calc block.